content/release-10-17-0/devices-10-17-0.md

```diff
@@ -77,6 +91,13 @@ Issue</th>
 <td>10.16.119.0</td>
 </tr>
 
+<tr>
+<td>Device Management</td>
+<td>The "Groups" column in the device list used to display all groups higher in the hierarchy to which the device belonged directly or transitively. This has been fixed and now only the direct parent groups are displayed.</td>
```

Comment:
```suggestion
<td>The "Groups" column in the device list formerly displayed all groups higher in the hierarchy to which the device belonged directly or transitively. This has been fixed and now only the direct parent groups are displayed.</td>
```
i suggest this because i had problems parsing the first sentence, not understanding how "used" is used here until the very end.